tests/integration/test_cogs_rates.py

```diff
@@ -21,6 +21,7 @@
 @pytest.fixture()
 def rates_cog(sample_bot):
     rates_cog = Rates(sample_bot)
+    rates_cog.rate_limit_delay = 0  # Override rate limit for testing. If we ever have tests fail due to this, we can adjust
```

Comment:
I've never seen tests fail due to rate limiting yet, so I'm keeping this fast for now. If we left it as default, then it just makes the integration tests have to wait which unnecessarily slows down test execution.